admin_solutions/index.adoc

```diff
@@ -0,0 +1,20 @@
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+The {product-title} Administrator Solutions guide is new as of version 3.2, and the topics cover the most common tasks faced by {product-title} administrators, with a focus on use cases and examples that guide the reader through each task.
+
+In this initial version of the guide, you can learn how to:
+
+- link:../admin_solutions/master_node_config.html[Configure masters and nodes using Ansible]
+- link:../admin_solutions/user_role_mgmt.html#limiting-and-monitoring-users-and-projects[Set limits for users and projects]
+- link:../admin_solutions/user_role_mgmt.html#determine-default-user-roles[Determine which roles users get by default]
+- link:../admin_solutions/user_role_mgmt.html#control-user-roles[Control User Permissions with Roles]
+- link:../admin_solutions/user_role_mgmt.html#share-templates-cluster[Share Templates Across the Cluster]
+- link:../admin_solutions/user_role_mgmt.html#create-cluster-admin[Create a Cluster Administrator]
+- link:../admin_solutions/authentication.html#github-auth[Configure GitHub as an Authentication Provider] (with more providers to follow in next release)
+
+Your feedback on this guide would be greatly appreciated. You can let us know if you find it to be helpful, or if there is a topic you would like us to cover, by https://github.com/openshift/openshift-docs/issues/new[filing an issue on GitHub].
```

Comment:
@tpoitras - best to point to openshift-docs@redhat.com instead of GitHub issues.